frontend/src/components/core/MainMenu/MainMenu.tsx

```diff
@@ -73,134 +75,197 @@ const getOpenInWindowCallback = (url: string) => (): void => {
   window.open(url, "_blank")
 }
 
-function MainMenu(props: Props): ReactElement {
-  const [dropdownOpen, setDropdownOpen] = useState(false)
-  const isServerDisconnected = !props.isServerConnected()
-
-  function toggleDropdown(): void {
-    setDropdownOpen(prevState => !prevState)
-  }
+export interface MenuListItemProps {
+  item: any
+  "aria-selected": boolean
+  onClick: (e: MouseEvent<HTMLLIElement>) => void
+  onMouseEnter: (e: MouseEvent<HTMLLIElement>) => void
+  $disabled: boolean
+  $isHighlighted: boolean
```

Comment:
The non-standard property names here are confusing w/o more context ("aria-selected" as a string literal, and $disabled/$isHighlighted with dollar prefixes) - can you add a small comment explaining why they're needed?

Reply:
Sigh...yes. That's BaseWeb's doing.